specs/approved/whole-system-configuration-steps.rst

```diff
@@ -92,75 +158,6 @@ The following are considered outside the scope of this specification:
 * Implementation of the approach by all hardware types; it is optional
 * Requiring a hardware type’s implementation be complete; it may be partial
 
-Solution
---------
-
-The solution adds three pairs of new, optional clean [0]_ and deploy [1]_
-steps. They are defined by the management hardware interface,
-``class ManagementInterface``.
-
-* ``export_configuration`` --
-          export a system's configuration and persistently store it as a
-          configuration mold, which can be used to configure other,
-          sufficiently capable systems. More details at
-          `Export configuration`_.
-* ``import_configuration`` --
-          import an existing configuration mold to a sufficiently capable
-          system in order to apply the configuration it describes to the
-          system. More details at `Import configuration`_.
-* ``import_export_configuration`` --
-          import and then export a system's configuration of the same system
-          as a single atomic step from the perspective of ironic's clean and
-          deploy mechanisms. This can further reduce the time required to
-          provision a system than a sequence of separate import and export
-          steps. More details at `Import and export configuration`_.
-
-Vendor independent configuration mold format is defined and example given in
-section `Format of configuration data`_. It contains a section for each
-hardware subsystem it supports. The desired configuration of a subsystem for
-which ironic has been offering configuration support is expressed as much as
-possible in ironic terms. For such subsystems, successful application of a
-configuration mold to an ironic node has the same side effects on the node's
-fields.
-
-This specification presently defines support for the BIOS and RAID subsystems
-ironic currently supports. The configuration of those is expressed in ironic
-terms. As part of applying BIOS and RAID configuration, the BIOS and RAID
-settings are cached.
-
-In the future, configuration mold format can be extended by adding support for
-more subsystems whose configurations can be expressed in a system-independent
-manner. Those could include boot, network, and firmware configuration, among
-others.
-
-To support provisioning beyond what ironic defines, but which systems and their
-hardware types could offer in a system-specific manner, a third section, OEM,
-is proposed. The OEM section can also be used to specify configuration ironic
-already supports. The content of the OEM section is specific to the system and
-its hardware type. Successful application of configuration specified in an OEM
-section has no side effects on the node's fields.
```

Comment:
I intentionally left both mentions of side effects, lines 122-124 and 140-141, out of the _Introduction_. They strike me as being too much detail.

As I continue my review, I will ensure they appear below, likely in the detailed discussion of _Import configuration_.

Reply:
going to add it to _Import configuration_ section.